README.md

```diff
@@ -3,6 +3,8 @@
 [![Slack Status](http://slack.citusdata.com/badge.svg)](https://slack.citusdata.com)
 [![Latest Docs](https://img.shields.io/badge/docs-latest-brightgreen.svg)](https://docs.citusdata.com/)
 [![Code Coverage](https://codecov.io/gh/citusdata/citus/branch/master/graph/badge.svg)](https://app.codecov.io/gh/citusdata/citus)
+[![Stack Overflow](https://img.shields.io/badge/Stack%20Overflow-%20-545353?logo=Stack%20Overflow)](https://stackoverflow.com/questions/tagged/citus)
+
```

Comment:
Could you drop this extra line and squash commits?

Reply:
Removed the extra line

Reply:
@metdos Could you check and approve for me to be able to merge the PR

Reply:
"squash commits?"

Could you do this first? I want to see only one commit.

Reply:
Squashed as you suggested